### DISCUSSION TITLE
A way to see who created which post?

### DISCUSSION THREAD

**Discussion by tom-sherman** [edmundhung/remix-guide#30] [Ideas]
Would be good for attribution and also safeguarding.

Someone posted a link to my blog, would be nice to know who to thank 😄

**Comment by tom-sherman** [comment]
I've just learned that remix-guide is editorialised, not user submitted content haha. So thank you remix-guide admins I guess!

**Comment by edmundhung** [comment]
> I've just learned that remix-guide is editorialised, not user submitted content haha. So thank you remix-guide admins I guess!

You are welcome and thanks for the great article too!

I made a bad decision early last year moving all the data to the cloud and then struggle releasing it to public because of the efforts need to monitor the platform from being abused. I had some plan to move the data back to GitHub so to make it easier for anyone to contribute but I had yet the time for the development yet. I hope I can get back to it later this year 🙂
